test/verify/check-machines

```diff
@@ -212,6 +212,8 @@ class TestMachines(MachineCase):
         b.wait_present("#test-vm-action-{0}".format(self.providerVm1_name)) # is provider-specific action button rendered?
         b.wait_in_text("#test-vm-props-{0}".format(self.providerVm1_name), "Test Provider Property") # are provider-specific VM props rendered?
 
+        b.click("{0}-sendNMI".format(self.providerVm1_id)) # click the Send Non-Maskable Interrupt button
```

Comment:
@sgratch , I'm sorry for not noticing that earlier - this check shall not be here in _testProvider_ since you did not implement such logic there.

The logic is implemented within the default provider (libvirt.es6 file), so move this test-step to either independent test or _testBasic_ . Considering complexity of this test, I would prefer the _testBasic_